src/entities/fractions/mod.rs

```diff
@@ -1,4 +1,11 @@
+/// This module contains functionality related to currency amounts in the context of fractions.
 pub mod currency_amount;
+/// This module contains functionality related to fractions, which are used for various calculations
+/// in the SDK.
 pub mod fraction;
+/// This module contains functionality related to percentages, which are used for expressing
+/// fractions in a more human-readable format.
 pub mod percent;
+/// This module contains functionality related to prices, which are used for calculating exchange
+/// rates and other financial metrics.
```

Comment:
These comments are so AI that they are too general and don't provide much insight.

Reply:
true, will still have to change them